src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -2183,6 +2183,21 @@ public void updateNetworkModificationsActivation(UUID studyUuid, UUID nodeUuid,
         notificationService.emitElementUpdated(studyUuid, userId);
     }
 
+    @Transactional
+    public void updateNetworkModificationsDescription(UUID studyUuid, UUID nodeUuid, UUID modificationUuid, String userId, String description) {
+        List<UUID> childrenUuids = networkModificationTreeService.getChildrenUuids(nodeUuid);
```

Comment:
I see this code is used in many others endpoint can we share it in a method ? if it is possible ?